uxarray/grid.py

```diff
@@ -308,7 +308,8 @@ def compute_face_areas(self, quadrature_rule="triangular", order=4):
             dim = self.Mesh2.attrs['topology_dimension']
 
             # initialize z
-            z = np.zeros((self.ds.nMesh2_node.size))
+            da_nMesh2_node = getattr(self.ds, self.ds_var_names['nMesh2_node'])
+            z = np.zeros((da_nMesh2_node.size))
```

Comment:
Could you please use the following code instead of these two lines, using the new functionality @philipc2 just implemented in PR #161 for this PR:

`z = np.zeros((self.nMesh2_node.size)) `

Reply:
OK, done.